internal/pkg/vm/internal/vmcontext/vmcontext.go

```diff
@@ -152,8 +151,9 @@ func (ctx *VMContext) LegacySend(to address.Address, method types.MethodID, valu
 		return nil, 1, errors.FaultErrorWrapf(err, "failed to get id address for actor address")
 	}
 
-	toActor, err := deps.GetOrCreateActor(context.TODO(), toAddr, func() (*actor.Actor, error) {
-		return &actor.Actor{}, nil
+	// TODO: this needs to initialize in init actor state
```

Comment:
Looks like this TODO is out of date.  The caller must have been init exec which does this bookkeeping already.